ui/src/components/ConnectionDataTable/index.jsx

```diff
@@ -37,6 +46,7 @@ export default class ConnectionDataTable extends Component {
     sortDirection: 'desc',
     headers: null,
     onHeaderClick: null,
+    withTopPagination: false,
```

Comment:
Now that I think about it, let's set to `true` and set `withTopPagination` to false in `TaskRunsCard/index.jsx` only.

Reply:
Applied changes!

Reply:
Is it possible you forgot to push your changes? :) Also, I think it would be better if we rename the prop to `withoutTopPagination` and keep the default value to `false`.

Reply:
Sorry about that! I've been trying to re-organize the commits and messages before I fully push the changes. It's all implemented now and the commits are also organized :)
Also, applied the changes to use `withoutTopPagination` as well!